src/Options/index.js

```diff
@@ -87,6 +87,9 @@ export interface ParseServerOptions {
   verbose: ?boolean;
   /* Sets the level for logs */
   logLevel: ?string;
+  /* Options for customizing log levels for specific events
+  :DEFAULT: {} */
+  logEvents: ?LogEventsOptions;
```

Comment:
I don't think this is needed; we already have a `logLevels` object to set the log levels for Cloud Code events, see https://github.com/parse-community/parse-server/issues/8529. I would just nest the new option there:

```js
{
  logLevels: {
    usernameAlreadyExists: 'silent'
  }
}
```

Reply:
What should be the options of usernameAlreadyExists? Is silent | error | info | verbose sound good?

Reply:
The same options as for the existing nested keys under logLevels. There shouldn't be a need to redefine that?